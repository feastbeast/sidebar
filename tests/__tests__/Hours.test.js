import React from 'react';
import Enzyme from 'enzyme';
import {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Hours from '../../client/src/components/Hours';

Enzyme.configure({ adapter: new Adapter() });



describe('Hours', () => {
  

  it('should be defined', () => {
    let hours = ['Open all day', 'Open all night'];

    let wrapper = shallow(<Hours
      hours={hours}
    />) 
    expect(wrapper).toBeDefined();
  });
  it('should render correctly', () => {
    let hours = ['Open all day', 'Open all night'];

    const tree = shallow(
      <Hours 
      hours={hours} 
      />
    );
    expect(tree).toMatchSnapshot();
  });
});