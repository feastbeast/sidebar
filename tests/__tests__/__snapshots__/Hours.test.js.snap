// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hours should render correctly 1`] = `
<Collapsible
  className=""
  classParentString="Collapsible"
  contentInnerClassName=""
  contentOuterClassName=""
  easing="linear"
  lazyRender={false}
  onClose={[Function]}
  onClosing={[Function]}
  onOpen={[Function]}
  onOpening={[Function]}
  open={false}
  openedClassName=""
  overflowWhenOpen="hidden"
  tabIndex={null}
  transitionCloseTime={null}
  transitionTime={400}
  trigger="Start here"
  triggerClassName=""
  triggerDisabled={false}
  triggerOpenedClassName=""
  triggerSibling={null}
  triggerStyle={null}
  triggerTagName="span"
>
  <p>
    This is the collapsible content. It can be any element or React component you like.
  </p>
  <p>
    It can even be another Collapsible component. Check out the next section!
  </p>
</Collapsible>
`;
